packages/wdio-cli/src/utils.ts

```diff
@@ -817,6 +815,22 @@ export async function npmInstall(parsedAnswers: ParsedAnswers, npmTag: string) {
     }
 }
 
+export type PM = 'npm' | 'yarn' | 'pnpm' | 'bun'
+export const PMs: PM[] = ['npm', 'yarn', 'pnpm', 'bun']
+/**
+ * detect the package manager that was used
+ */
+export function detectPackageManager() {
```

Comment:
Make this easier to unit test via:

```suggestion
export function detectPackageManager(argv = process.argv) {
```